starter-project-web-react/pages/index.tsx

```diff
@@ -1,24 +1,52 @@
 import type { NextPage } from 'next'
-import Bloglist from './blogs'
-import { getSession } from 'next-auth/react'
-import axios from 'axios'
+import { Box, Grid, Typography } from '@mui/material'
+import Link from 'next/link'
 
 const Home: NextPage = () => {
-  return <Bloglist />
+  const personalPage = ['mintesnot']
+  return (
+    <Box
+      sx={{
+        mx: 'auto',
+        pt: '20px',
+        boxShadow: 3,
+        maxWidth: '60%',
+        minHeight: '92vh',
+      }}
+    >
+      <Grid container spacing={0}>
+        {personalPage.map((name, idx) => {
+          return (
+            <Grid key={idx} item xs={12}>
+              <Link
+                href={{
+                  pathname: '${name}/',
+                }}
+                passHref
+                as={`${name}/`}
+              >
+                <Box
+                  sx={{
+                    my: '5px',
+                    height: '80px',
+                    maxWidth: '96%',
+                    boxShadow: 3,
+                    mx: 'auto',
+                    display: 'flex',
+                    alignItems: 'center',
+                    px: '5px',
+                  }}
+                >
+                  <Typography variant="h3" sx={{ fontWeight: 'bold' }}>
+                    Mintesnot
```

Comment:
should be dynamic it is better to use object array like this [{name: "mintesnot", link: "/mintesnot"}]